docs/02-app/01-building-your-application/03-rendering/01-server-components.mdx

```diff
@@ -0,0 +1,122 @@
+---
+title: Server Components
+description: Learn how you can use React Server Components to render parts of your application on the server.
+related:
+  description: Learn how Next.js caches data and the result of static rendering.
+  links:
+    - app/building-your-application/caching
+---
+
+React Server Components allow us to write UI that can be rendered and optionally cached on the server. In Next.js, the rendering work is further split by route segments to enable streaming and partial rendering, and there are three different server rendering strategies:
```

Comment:
```suggestion
React Server Components allow you to write UI that can be rendered and optionally cached on the server. In Next.js, the rendering work is further split by route segments to enable streaming and partial rendering, and there are three different server rendering strategies:
```

Correct me if I am wrong, but I believe we try and avoid "us" and "we" in favor of "you".

Reply:
Yes, my brain thinks in "we" 🙈 thank you!